src/runtime_src/core/common/utils.cpp

```diff
@@ -259,4 +260,44 @@ parse_clock_id(const std::string& id)
   return clock_str != clock_map.end() ? clock_str->second : "N/A";
 }
 
+uint64_t
+mac_addr_to_value(std::string mac_addr)
+{
+  boost::erase_all(mac_addr, ":");
+  // Convert the reduced string into a value using the hex formatter
+  std::istringstream converter(mac_addr);
```

Comment:
This conversion does work.  It can be simplified as follows:
```
uint64_t
mac_addr_to_value(std::string mac_addr)
{
  boost::erase_all(mac_addr, ":");
  uint64_t mac_addr_value = std::stoll(formattedOutput, nullptr, 16);
  return mac_add_value;
}
```

Reply:
Super neat! I used std::stoul though to prevent any casting issues.